testing/cloud_layouts/EKS/WithoutNAT/configuration.tpl.yaml

```diff
@@ -71,7 +71,7 @@ spec:
   settings:
     modules:
       publicDomainTemplate: "%s.k8s.smoke.flant.com"
-    storageClass: localpath-all
+      storageClass: localpath-all
   version: 1
```

Comment:
version should be changed to 2, I suppose

Reply:
thank you. done